linear_search.cpp

```diff
@@ -0,0 +1,40 @@
+//HEADER FILES
+#include<iostream>
```

Comment:
This include needs to removed as iostream is already included in **bits/stdc++.h**